### PR TITLE
[hummus-recipe] Add missing RecipeOptions

### PR DESCRIPTION
**Pull request by Maerig**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://hummus-recipe.s3.amazonaws.com/docs/Recipe.html
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/hummus-recipe/index.d.ts [hunk 1]
```diff
@@ -55,7 +55,13 @@ declare namespace Recipe {
         author?: string;
         title?: string;
         subject?: string;
+        colorspace?: string;
```
Comment (she11sh0cked) [inline types/hummus-recipe/index.d.ts:58]: This should be `colorspace?: 'rgb' | 'cmyk' | 'gray';` right?

**Review by she11sh0cked — CHANGES_REQUESTED** [review]

File: types/hummus-recipe/index.d.ts (hunk 1 above)
Comment (Maerig) [reply types/hummus-recipe/index.d.ts:58]: You're right. I've checked tests in hummusRecipe just to make sure: https://github.com/chunyenHuang/hummusRecipe/blob/master/tests/color.js

**Comment by Maerig** [comment]
@she11sh0cked @sebastinez I've updated my changes.

**Review by peterblazejewicz — APPROVED** [review]
LGMT!
@Maerig thx!

**Comment by Maerig** [comment]
Ready to merge, thanks @peterblazejewicz!

**Pull request merged**